java/src/org/openqa/selenium/grid/config/TomlConfig.java

```diff
@@ -19,30 +19,27 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSortedSet;
-import io.ous.jtoml.JToml;
-import io.ous.jtoml.ParseException;
-import io.ous.jtoml.Toml;
-import io.ous.jtoml.TomlTable;
 import java.io.IOException;
 import java.io.Reader;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
I am not sure if wild card imports is a practice in the codebase. Can we revert back to explicit imports ( I know that perhaps the IDE does this)

Reply:
I will remove this when I complete the PR and pass the tests.